Sources/Fluent/Entity/Entity.swift

```diff
@@ -138,11 +152,11 @@ extension Entity {
         Finds the entity with the given `id`.
     */
     public static func find(_ id: NodeRepresentable) throws -> Self? {
-        guard let idKey = database?.driver.idKey else {
+        if(database == nil) {
```

Comment:
Let's change this to `guard let _ = database else { return nil }` (in general, avoid parens to stick w/ our style 👍 )